tests/core/test_actions.py

```diff
@@ -216,6 +216,52 @@ def test_domain_action_instantiation():
     assert events[2] == SlotSet("name", "rasa")
 
 
+async def test_remote_action_utterances_with_None_values(
+    default_channel, default_tracker, default_domain
+):
+    endpoint = EndpointConfig("https://example.com/webhooks/actions")
+    remote_action = action.RemoteAction("my_action", endpoint)
+
+    response = {
+        "events": [
+            {"event": "form", "name": "restaurant_form", "timestamp": None},
+            {
+                "event": "slot",
+                "timestamp": None,
+                "name": "requested_slot",
+                "value": "cuisine",
+            },
+        ],
+        "responses": [
+            {
+                "text": None,
+                "buttons": None,
+                "elements": [],
+                "custom": None,
+                "template": "utter_ask_cuisine",
+                "image": None,
+                "attachment": None,
+            }
+        ],
+    }
+
+    nlg = TemplatedNaturalLanguageGenerator(
+        {"utter_ask_cuisine": [{"text": "what dou want to eat?"}]}
+    )
+    with aioresponses() as mocked:
+        mocked.post("https://example.com/webhooks/actions", payload=response)
+
+        events = await remote_action.run(
+            default_channel, nlg, default_tracker, default_domain
+        )
+
+    assert events == [
+        BotUttered("what dou want to eat?", data={"buttons": [], "elements": []}),
```

Comment:
Is this normal, or are the buttons and elements not getting cleaned correctly (i think everything else disappears if it is `None`)?